build_modules/build.yaml

```diff
@@ -1,6 +1,15 @@
 builders:
+  meta_module:
```

Comment:
Now that I'm looking at it, I wonder if we should make this a single builder definition from the start...

I'd merge this with the `modules` builder

Reply:
Leaving for a follow up PR.